app/views/restaurants/_form.html.erb

```diff
@@ -1,65 +1,65 @@
 <%= form_for @restaurant, :html => { :multipart => true } do |f| %>
+  <% if @restaurant.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@restaurant.errors.count, "error") %> prohibited this restaurant from being saved:</h2>
 
```

Comment:
this hidden field are from poylmer???
Add a line note
  <%= hidden_fields(f, :name, :description, :slogan, :phone_number, :email, :address, :twitter, :instagram, :facebook, :website, :delivery) %>